source/common/upstream/outlier_detection_impl.cc

```diff
@@ -330,7 +348,8 @@ void DetectorImpl::runCallbacks(HostSharedPtr host) {
   }
 }
 
-void EventLoggerImpl::logEject(HostDescriptionConstSharedPtr host, EjectionType type) {
+void EventLoggerImpl::logEject(HostDescriptionConstSharedPtr host, EjectionType type,
```

Comment:
Wouldn't it be useful to log some of the SR stats here also to see the SR, the cutoff, etc.?